### PR TITLE
*: setup CI

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: Makefile.common [hunk 1]
```diff
@@ -0,0 +1,301 @@
+# Copyright 2018 The Prometheus Authors
```
Comment (paulfantom) [inline Makefile.common:1]: 2020?

Comment (simonpasquier) [reply Makefile.common:1]: That file comes from https://github.com/prometheus/prometheus and will automatically updated whenever the original file changes (based on your amazing script :-p)

Comment (paulfantom) [reply Makefile.common:1]: Seems we will be at 2018 forever :smile:

**Comment by paulfantom** [comment]
I've enabled branch protection and now following checks are necessary:
- DCO
- ci/circleci: build
- ci/circleci: test

**Pull request merged**